### DISCUSSION TITLE
Discover Region - Allow Multiple Countries

### DISCUSSION THREAD

**Discussion by mranderson256** [sct/overseerr#3145] [Feature Requests]
It would be great to be able to only select US and UK in order to exclude all other regions.

The same way you can select multiple languages.

**Comment by donato** [comment]
Agreed!

I'm using the region locking feature to limit the amount of adult content that is incorrectly tagged and showing up in discover feeds.

**Comment by EmSixTeen** [comment]
Why on earth is it limited to just one region?

**Comment by RChesterton** [comment]
Adding to this, I'd love to be able to select more than one region

**Comment by skeptile2** [comment]
I'm using it for the same reason as donato above - limiting adult content - and would love to be able to select multiple.

**Comment by Valtora** [comment]
Just submitted a PR to enable this feature. Works quite nicely in my dev environment. Let's see how it goes and if it gets merged into master.

<img width="1637" height="1225" alt="Screenshot 2025-12-28 113452" src="https://github.com/user-attachments/assets/eba07b4d-d3ce-499a-9f98-c0163bcd4bbf" />

**Reply by EmSixTeen** [reply]
Awesome, hopefully gets merged in!

**Reply by Valtora** [reply]
Fingers crossed, FYI I've closed the PR on this repo because the team is moving across to Seerr with Jellyseerr and Overseerr merging to become just Seerr. 

Link to PR: https://github.com/seerr-team/seerr/pull/2260
